docs/push_server.rst

```diff
@@ -192,6 +192,29 @@ For example, on Zigbee sub-devices you also need to define `source_sid` and `sou
 see :ref:`button press <_button_press_example>` for an example.
 See the :class:`PushServer.EventInfo` for more detailed documentation.
 
+Implement a gateway zigbee device
+--------
+
+Once the event information is obtained as discribed above a new event for a zigbee device connected to a gateway can be implemented as follows:
+
+1. go to the miio\gateway\devices\subdevices.YAML file of this python module and search for the device you want to implement the event for.
+2. Add the event you just discoved like this:
```

Comment:
```suggestion
2. Add an entry for the :ref:`newly discovered event<obtain_event_info>`_:
```
This will create a link to the correct place in the push server docs (I hope).